import unit ontology

akleinau opened the pull request:
closes #12

This imports uo without the PATO classes, puts its classes in immaterial entity where UnitOfMeasurement was before and deletes that class.

I'm not sure about the whole import process so I hope it works the way I planned it.

akleinau commented:
also I imported here uo into oeo because this made sense from the folder structure (oeo and the two folders edits and imports that get imported) but maybe it makes more sense to import uo, ro, geo etc into the appropriate modules? E.g. uo in oeo-physical?

jannahastings commented:
I think the import into oeo.omn works as long as we do not need to refer to units in the axioms in any module. However, to import it in the modules should not create a problem either. It will then be available in the full ontology transitively.

jannahastings commented:
There are some outstanding problems:
- There is still a duplicate openenergy class Unit with no definition (subclass of ModelVariable)
- some individuals representing units (e.g. Ampere) still remain in the ontology and need to be removed

akleinau commented:
> There are some outstanding problems:
> 
> * There is still a duplicate openenergy class Unit with no definition (subclass of ModelVariable)
> * some individuals representing units (e.g. Ampere) still remain in the ontology and need to be removed

I fixed those and changed the importing file to oeo-physical instead of oeo because I can imagine needing units in axioms. I hope I got the formulation of the relative URI right so it doesnt just work for me :)

l-emele commented:
I have no idea how imports work so I refrain from reviewing this pull request.
